[ADD] accounting: localization page for uk (#3230)
odoo/documentation: content/applications/finance/accounting/fiscal_localizations/localizations/united_kingdom.rst

==============
United Kingdom
==============

Configuration
=============

:ref:`Install <general/install>` the :guilabel:`UK - Accounting` and the :guilabel:`UK - Accounting
Reports` modules to get all the features of the UK localization.

.. list-table::
   :header-rows: 1

   * - Name
     - Technical name
     - Description
   * - :guilabel:`UK - Accounting`
     - `l10n_uk`
     -  - CT600-ready chart of accounts
        - VAT100-ready tax structure
        - Infologic UK counties listing
   * - :guilabel:`UK - Accounting Reports`
     - `l10n_uk_reports`
     -  - Accounting reports for the UK
        - Allows sending the tax report via the MTD-VAT API to HMRC.

> .. image:: united-kingdom/uk.png

Reviewer: Use the same name as the document file for resource folders.
```suggestion
.. image:: united_kingdom/uk.png
```

> .. image:: united-kingdom/uk.png

Reviewer: The image is added twice as `uk.png` and `uk-fs8.png`.

   :align: center
   :alt: Odoo uk packages

.. note::
   - Only UK-based companies can submit reports to HMRC.
   - Installing the module :guilabel:`UK - Accounting Reports` installs all two modules at once.

.. seealso::
   - `HM Revenue & Customs <https://www.gov.uk/government/organisations/hm-revenue-customs/>`_

>    - `Overview of Making Tax Digital <https://www.gov.uk/government/publications/making-tax-digital/overview-of-making-tax-digital/>`_

Reviewer: ```suggestion
   - `Overview of Making Tax Digital
     <https://www.gov.uk/government/publications/making-tax-digital/overview-of-making-tax-digital/>`_
```

Chart of accounts
=================

The UK chart of accounts is included in the :guilabel:`UK - Accounting` module. Go to
:menuselection:`Accounting --> Configuration --> Accounting: Chart of Accounts` to access it.

Setup your :abbr:`CoA (chart of accounts)` by going to :menuselection:`Accounting --> Configuration
--> Settings --> Accounting Import section` and choose to :guilabel:`Review Manually` or
:guilabel:`Import (recommended)` your initial balances.

Taxes
=====

As part of the localization module, UK taxes are created automatically with their related financial
accounts and configuration.

Go to :menuselection:`Accounting --> Configuration --> Settings --> Taxes` to update the
:guilabel:`Default Taxes`, the :guilabel:`Tax Return Periodicity` or to :guilabel:`Configure your
tax accounts`.

To edit existing taxes or to :guilabel:`Create` a new tax, go to :menuselection:`Accounting -->
Configuration --> Accounting: Taxes`.

.. seealso::
   - :ref:`Taxes <Taxes>`

>    - Tutorial: `Tax report and return <https://www.odoo.com/slides/slide/tax-report-and-return-1719?fullscreen=1>`_.

Reviewer: ```suggestion
   - Tutorial: `Tax report and return
     <https://www.odoo.com/slides/slide/tax-report-and-return-1719?fullscreen=1>`_.
```

Making Tax Digital (MTD)
------------------------

In the UK, all VAT-registered businesses are required to follow the MTD rules by using software to
submit their VAT returns.

The **UK - Accounting Reports** module enables you to comply with the `HM Revenue & Customs
<https://www.gov.uk/government/organisations/hm-revenue-customs/>`_ requirements regarding

> `Making Tax Digital <https://www.gov.uk/government/publications/making-tax-digital/overview-of-making-tax-digital/>`_ in the UK.

Reviewer: ```suggestion
`Making Tax Digital <https://www.gov.uk/government/publications/making-tax-digital/overview-of-making-tax-digital/>`_
in the UK.
```

.. important::
   If your periodic submission is more than three months late, it is no longer possible to submit
   your report through Odoo. Odoo only retrieves open bonds from the last three months. Your
   submission has to be done manually by contacting HMRC.

Register your company to HMRC before the first submission

> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Reviewer: Use `~` for H4 delimiters.

        - Infologic UK counties listing
   * - :guilabel:`UK - Accounting Reports`
     - `l10n_uk_reports`
     -  - Accounting reports for the UK
        - Allows sending the tax report via the MTD-VAT API to HMRC.

.. image:: united-kingdom/uk.png
   :align: center
   :alt: Odoo uk packages

.. note::
   Only UK-based companies can submit reports to HMRC.

.. seealso::
   - `HM Revenue & Customs <https://www.gov.uk/government/organisations/hm-revenue-customs/>`_
   - `Overview of Making Tax Digital <https://www.gov.uk/government/publications/making-tax-digital/overview-of-making-tax-digital/>`_

Chart of accounts
=================

The chart of accounts is installed by default as part of the set of data included in the
:guilabel:`UK - Accounting` module. You can find it by going to :menuselection:`Accounting -->
Configuration --> Accounting: Chart of Accounts`.

Choose how you want to setup your :abbr:`CoA (chart of accounts)` by going to
:menuselection:`Accounting --> Configuration --> Settings --> Accounting Import section`. You can
:guilabel:`Review Manually (end of year balance)` or :guilabel:`Import (for full history)` your
initial balances.

> We recommend importing your initial balances using the chart of account import.

Author: good suggestion thanks

==============
United Kingdom
==============

Configuration
=============

:ref:`Install <general/install>` the following modules to get all the features of the UK
localization:

.. list-table::
   :header-rows: 1

   * - Name
     - Technical name
     - Description
   * - :guilabel:`UK - Accounting`
     - `l10n_uk`
     - Installing this module grants you access to:
        - CT600-ready chart of accounts
        - VAT100-ready tax structure

>         - InfoLogic UK counties listing

Reviewer 2: ```suggestion
        - InfoLogic UK counties listing
```
Couldn't find anything online on "InfoLogic" on Google, however I found something under "ListLogic", which *seems* to list counties in the UK. Might be worth checking with whoever is responsible

Author: Info to be confirmed. In the meantime, let's keep it as such.

>         - a few other adaptations

Author: I didn't find release notes covering UK. I found the "and a few other adaptations" in the module description. Will remove it.

   * - :guilabel:`UK - Accounting Reports`
     - `l10n_uk_reports`
     - Installing this module grants you access to:
        - Accounting reports for UK
        - Allows sending the tax report via the MTD-VAT API to HMRC.

.. tip::

>    Installing the module :guilabel:`UK - Accounting Reports` installs all two modules at

Author: good suggestion.

   once.

.. note::
   Your company should be located in the UK to be able to submit HMRC reports.

.. seealso::
   - `HM Revenue & Customs <https://www.gov.uk/government/organisations/hm-revenue-customs/>`_
   - `Overview of Making Tax Digital <https://www.gov.uk/government/publications/making-tax-digital/overview-of-making-tax-digital/>`_

> 

Reviewer 2: ```suggestion

```
Might be worth adding a section here on the UK - Accounting module, and more specifically *what* it includes. Since in the task people seem to ask for screenshots, you could maybe add a chart of accounts of a UK local, as well as the tax structure (and where to find them)
![uk-accounts](https://user-images.githubusercontent.com/104567387/209557765-c7c10a9e-e344-456c-9a64-95e6208a4d71.png)

Author: Updated. New sections added.

Making Tax Digital (MTD)
========================

> Making Tax Digital is a key part of the UK government’s plans to make it easier for individuals and

Reviewer 2: ```suggestion
Making Tax Digital is a key part of the UK government’s plans to make it easier for individuals and
```
I think users looking for the UK localization already know what MTD is, and Odoo isn't the place for that information. IMO it is better to keep it straight to the point, the line "All VAT-registered businesses..." is much more relevant to the doc :)

Author: make sens!

> Making Tax Digital is a key part of the UK government’s plans to make it easier for individuals and

Reviewer 2: ```suggestion
Making Tax Digital (MTD) is a key part of the UK government’s plans to make it easier for individuals and
```

businesses to get their tax right and keep on top of their affairs. All VAT-registered businesses

> are required to follow the MTD rules by keeping digital records and using software to submit their

Reviewer 2: ```suggestion
are required to follow the MTD rules by keeping digital records and using software to submit their
```
Records of what ? I'm guessing VAT transactions

VAT returns.

> The **UK - Accounting Reports** package enables you to comply with the `HM Revenue & Customs <https://www.gov.uk/government/organisations/hm-revenue-customs/>`_

Reviewer 2: ```suggestion
The **UK - Accounting Reports** module enables you to comply with the `HM Revenue & Customs <https://www.gov.uk/government/organisations/hm-revenue-customs/>`_
```

requirements regarding `Making Tax Digital <https://www.gov.uk/government/publications/making-tax-digital/overview-of-making-tax-digital/>`_ in the UK.

> .. important::

Reviewer 2: ```suggestion
.. important::
```
I'll let you decide, but this part *seems to me* about VAT return submission, which is nowhere to be found in the doc. Access as well as registering to HMRC are explained, but not how to submit. If I remember correctly, it is explained in the video "Tax report" with Eva, otherwise you can always find the script in my deployed tasks

Author: This is now part of the Taxes section.
The submission part is already explained in the next section.
I added a see also to the tutorial, good idea. However

   If your periodic submission is more than three months late, it is no longer possible to submit
   your report through Odoo. Odoo only retrieves open bonds from the last three months. Your
   submission has to be done manually by contacting HMRC.

Register your company to HMRC before the first submission
---------------------------------------------------------

Import your obligations HMRC, filter on the period you want to submit and send the VAT Return report
by clicking on :guilabel:`Send to HMRC`.

.. note::

>    Configure your Tax Return Periodicity by going to :menuselection:`Accounting Settings -->

Reviewer 2: ```suggestion
   Configure your Tax Return Periodicity by going to :menuselection:`Accounting Settings -->
```
This should be part of the "how to submit your tax report" section, and not as a note

Author: moved to line 59-60

   Taxes section`. Click :guilabel:`Configure your tax accounts` to update your Tax groups.

Periodic submission to HMRC
---------------------------

> Only one company - and one user - can connect to HMRC simultaneously. If several UK companies are on

Reviewer 2: ```suggestion
Only one company - and one user - can connect to HMRC simultaneously. If several UK-based companies are on
```

> the same database, the user that submits the HMRC report follows these instructions each time he

Reviewer 2: ```suggestion
the same database, the user that submits the HMRC report has to  follow these instructions each time they
```
"They" is neutral and preferred in English

wants to proceed to a submission:

#. Log into the company for which the submission has to be done.
#. Go to :guilabel:`Settings app` and in the :guilabel:`Users` section click

>    :guilabel:`Manage Users`. Select the user who will submit VAT.

Reviewer 2: ```suggestion
   :guilabel:`Manage Users`. Select the user who will submit the VAT return.
```

#. Go to :guilabel:`UK HMRC Integration` tab and click :guilabel:`Reset Authentication Credentials`
   or :guilabel:`Remove Authentication Credentials` button. Then click again to create new
   credentials. It is now possible to submit the tax report for this company.

> #.  Repeat the steps to proceed to the HMRC submission for another company.

Reviewer 2: ```suggestion
#.  Repeat the steps to proceed to the HMRC submission for another company.
```
Repeat the steps for other companies' HMRC submissions.

.. note::

>    During this process, the :guilabel:`Connect to HMRC` button no longer appears for other UK

Reviewer 2: ```suggestion
   During this process, the :guilabel:`Connect to HMRC` button no longer appears for other UK-based
```
